### PR TITLE
Add missing documentation for st2scheduler

### PR DESCRIPTION
**Pull request by armab**
Copy missing `scheduler` documentation from st2docs https://docs.stackstorm.com/install/k8s_ha.html#st2scheduler to `stackstorm-ha`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,8 @@ All the workflow engine processes will share the load and pick up more work if o
 >  [Orquesta st2workflowengine](https://docs.stackstorm.com/orchestra/index.html) as a new native workflow engine.
 
 ### [st2scheduler](https://docs.stackstorm.com/reference/ha.html#st2scheduler)
-TODO: Description TBD
+`st2scheduler` is responsible for handling ingress action execution requests.
+`2` replicas for K8s Deployment are configured by default to increase StackStorm scheduling throughput.
```
Comment (warrenvw) [inline README.md]: Maybe start with a word instead? I suggest: "By default, 2 replicas are configured in the K8s deployment to increase StackStorm scheduling throughput."

Comment (armab) [reply README.md]: Seems very minor.

The issue is that it's extracted from the https://docs.stackstorm.com/install/k8s_ha.html so I'm trying to keep both docs in sync.
Any wording change would need to be done in both places.

Comment (warrenvw) [reply README.md]: Yes, it's definitely minor. I am okay leaving it as is if you'd prefer.

Comment (armab) [reply README.md]: Thanks, merging then!

**Pull request merged**